packages/lodestar/src/network/error.ts

```diff
@@ -11,3 +13,19 @@ export class RpcError extends Error {
     this.status = status;
   }
 }
+
+export const RESPONSE_TIMEOUT_ERR = "response timeout";
+
+export function isResponseTimeout(err: Error): boolean {
+  return err.message.includes(RESPONSE_TIMEOUT_ERR);
+}
+
+export function updateBlockProviderErrorScore(scoreTracker: IRpcScoreTracker, peer: PeerId, e: Error): void {
```

Comment:
one last reference to block provider, `updateRpcScore`?

Reply:
dammit